### ISSUE TITLE
Conflict handling screen: Link to involved object not working 

### ISSUE THREAD

**Issue by sun-geo** [openstreetmap/iD#10048]
### URL

Probably everywhere where a conflict is created 

### How to reproduce the issue?

- download an existing osm object at the same time with 2  different user accounts
- edit the object with first account and upload the data
- edit the object with a second account and try to upload the data
- during this saving situation the error message of a conflict should appear
- in the conflict dialog there is an section where the involved object is listed (in screenshot the blue text "3711" object)
- click on the object to show the object on osm.org
- instead of showing/linking (to) the object details it sends you to an default osm.org page without object information

So for my understanding to solve this issue, there are 2 options:
- either don't show the object name in blue text (so there is no intention that this is a link)
- or fix the link function

(Don't wonder, object "3711" and user "Barew" in the screenshot are anonymized)

### Screenshot(s) or anything else?

![conflict while upload an](https://github.com/openstreetmap/iD/assets/47290585/d97889e8-7931-44c0-ada4-8a70a7fa49ed)

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

current v2.27.3

### Which browsers are you seeing this problem on?

Firefox

**Comment by tyrasd** [comment]
yeah, this seems like a small oversight in how the link is generated. also, it seems like the link's text could be made a bit more readable (e.g. `way 3711`)
